connector/saml/saml.go

```diff
@@ -81,8 +81,10 @@ type Config struct {
 	//
 	// https://www.oasis-open.org/committees/download.php/35391/sstc-saml-metadata-errata-2.0-wd-04-diff.pdf
 
-	Issuer string `json:"issuer"`
-	SSOURL string `json:"ssoURL"`
+	// Issuer refers to the SP issuer
+	Issuer    string `json:"issuer"`
+	SSOIssuer string `json:"ssoIssuer"`
```

Comment:
Sorry, we've been dealing with a security issue in this connector. Please see https://groups.google.com/forum/#!topic/coreos-user/s_BWh4a3E1Y